### ISSUE TITLE
Render is being called twice

### ISSUE THREAD

**Issue by hwangar** [isaachinman/next-i18next#433]
I noticed this behavior after researching on a curious flickering driving me nuts that ended up in next-i18next code

You can reproduce it just adding a console.log in the "simple example"'s pages/index.js at render method: if appWithTranslation HOC is present, it's called twice, else, just one

I even disabled localeSubpaths: 'none' but it didn't help, just curious about if there's a reason behind or it's a bug

BTW, thank you very much for your awesome work

Juan Arias

**Comment by mslepcevic** [comment]
Looks like this is happening to me too if I upgrade to any version above 0.50.0 (currently using next 9.0.3, but was also happening on previous 9 releases).

Unfortunately, I cannot get a reproducible repo going, because the project I have is private and a big one, and I'm not sure what exactly seems to be going on.

I basically have some API routes which are used to provide some data from the server to the client, while SSR has direct access to the same data. The initial render triggers no API calls with <=0.50.0, since it can fetch the data itself during the SSR. If I upgrade, my pages get the usual SSR treatment, but after they show up, the API calls also get triggered. It's as if there's a new client side render trigger once the SSR result is shown.

Version 0.51.0 seems to trigger the client side re-fetch twice once SSR is done (seems like there might be 3 renders called there), while 0.51.1 and above do it only once.

**Comment by isaachinman** [comment]
Probably related to https://github.com/isaachinman/next-i18next/commit/d7623f6bc9a37dd8aea5fbd9d9838697968e70b0. Someone want to take a look and let me know?

**Comment by isaachinman** [comment]
@hwangar @mslepcevic Can you please check/review #437?

**Comment by hwangar** [comment]
@isaachinman thank you very much!, I checked the rollback and it works for me
in my tomorrow I'll check your fix

**Comment by mslepcevic** [comment]
@isaachinman I used the changes from #437 locally, and the unneeded API calls seem to no longer be there, so looks good from my end.
💪 💪 💪

**Comment by Druux** [comment]
@isaachinman I am still dealing with this issue. I just updated to the latest release. Did I miss something?

**Comment by isaachinman** [comment]
@Druux Please provide a reproducible example in a new issue.
